src/main/java/com/marcosoft/kartracingapi/repository/PilotEntityRepository.java

```diff
@@ -2,6 +2,9 @@
 
 import com.marcosoft.kartracingapi.entity.PilotEntity;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.rest.core.annotation.RestResource;
+
+import javax.annotation.Resource;
 
 public interface PilotEntityRepository extends CrudRepository<PilotEntity,Integer> {
```

Comment:
sugestion: Change this and all other repositories to JpaRepository to avoid casting to Lint when callign findAll()